### DISCUSSION TITLE
Using index data from multiple specific countries

### DISCUSSION THREAD

**Discussion by manuelwaldner** [komoot/photon#721] [Q&A]
Hi,

we currently only use the extracted db data for germany, but now want to expand our data to also include austria.
I couldn't find a way to combine two extracted db sets into one, so here i am.
What i'm trying to do specifically is to use both of the latest tars from:
https://download1.graphhopper.com/public/extracts/by-country-code/at/
https://download1.graphhopper.com/public/extracts/by-country-code/de/

And make our photon instance return results for e.g. Wien (Austria) and Berlin (Germany) but no results from e.g. Spain etc.

Thank you 🙏🏻

**Comment by lonvia** [comment]
There is currently no way to merge the country extracts that we provide. You need to run your own export from Nominatim if you need other regions.
